src/System.Private.CoreLib/src/System/Diagnostics/Eventing/EventPipeController.cs

```diff
@@ -37,38 +37,56 @@ internal sealed class EventPipeController
         private const int EnabledPollingIntervalMilliseconds = 1000; // 1 second
         private const int DisabledPollingIntervalMilliseconds = 5000; // 5 seconds
         private const uint DefaultCircularBufferMB = 1024; // 1 GB
-        private static readonly char[] ProviderConfigDelimiter = new char[] { ',' };
-        private static readonly char[] ConfigComponentDelimiter = new char[] { ':' };
-        private static readonly string[] ConfigFileLineDelimiters = new string[] { "\r\n", "\n" };
         private const char ConfigEntryDelimiter = '=';
 
+        private static char[] ProviderConfigDelimiter => Defaults.ProviderConfigDelimiter;
```

Comment:
This should better use the Split overloads that take `char` to avoid the allocation altogether.

Reply:
Done